jupyterlab_gitsync/setup.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2020 Google LLC
```

Comment:
are we putting a python package and JS package in the same root folder?

I suggest we have good isolation between these two packages. Probably create a folder for UI extension and another for the backend handler.

Reply:
This is automatically generated by cookiecutter. All other extensions are formatted this way.